guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -203,12 +209,8 @@ This image is generic to all hosts with a provisioning NIC on the same subnet.
 For more information about configuring security tokens, read xref:Provisioning_Bare_Metal_Hosts-Security_Token_in_the_Boot_Process[].
 
 [NOTE]
-ifdef::satellite[]
-The *Full host image* is based on SYSLINUX and works with all {RHEL} certified hardware.
-endif::[]
-ifndef::satellite[]
 The *Full host image* is based on SYSLINUX and works with most hardware.
```

Comment:
```suggestion
The *Full host image* is based on SYSLINUX and Grub and works with most hardware.
```

For satellite context, you can add: This image type is supported on all RHEL supported hardware. Or something like that.

Reply:
This section is excluded from satellite.

Reply:
The section Creating Hosts with PXE-less Provisioning is removed from satellite. 173-315

Reply:
Not sure what you mean, this hunk we are discussing should stay. There is "if satellite" because it talks RHEL certifications which are irrelevant upstream.

So just keep it and add "Grub" as well.

You can drop my "This image type is supported on all RHEL supported hardware" idea, that's fine.